resources/migrations/custom_sql/fill_cache_config.my.sql

```diff
@@ -4,8 +4,8 @@ WITH root AS (
          0      AS model_id,
          'ttl'  AS strategy,
          json_object(
-           'multiplier', coalesce((select cast(`value` as unsigned) from setting where `key` = 'query-caching-ttl-ratio'), 10),
-           'min_duration_ms', coalesce((select cast(`value` as unsigned) from setting where `key` = 'query-caching-min-ttl'), 60000)
+           'multiplier', coalesce((select cast(cast(`value` as float) as unsigned) from setting where `key` = 'query-caching-ttl-ratio'), 10),
+           'min_duration_ms', coalesce((select cast(cast(`value` as float) as unsigned) from setting where `key` = 'query-caching-min-ttl'), 60000)
```

Comment:
Minor, but I think a ceiling is safer than a truncation for this case.

Also, this expression is failing for MariaDB, but I'm not sure why.

Reply:
ceiled all up